userspace/otpilot/src/main.rs

```diff
@@ -183,9 +199,24 @@ fn run() -> TockResult<()> {
             console_reader::get().allow_read(1)?;
         }
 
+        if gpio::get().have_events() {
+            match gpio_processor.process_gpio_events() {
+                Ok(()) => {}
+                Err(_) => {
+                    // Ignore error from writeln. There's nothing we can do here anyway.
+                    let _ = writeln!(console, "Device: Error processing GPIO event.");
```

Comment:
Is it possible to add any detail about the event for which processing failed?

Reply:
I created #227 to generally implement better handling in these places.